src/content/doc-surrealdb/deployment/surreal cloud.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 1
+sidebar_label: Deploy on Surreal Cloud (Beta)
+title: Deploy on Surreal Cloud (Beta) | Deployment
+description: This tutorial guides users through the process of deploying SurrealDB using Surreal Cloud.
+---
+
+import Image from "@components/Image.astro";
+
+import NamespaceDatabaseLight from '@img/image/cloud/namespace-database.png';
+import NamespaceDatabaseDark from '@img/image/cloud/namespace-database.png';
+
+import LightCloud from '@img/image/cloud/surrealcloud.png';
+import DarkCloud from '@img/image/cloud/surrealcloud.png';
+
+import OpenInSDK from '@img/image/cloud/open-in-sdk.png';
+
+import QueryingInstanceLight from '@img/image/cloud/querying-instance.png';
+import QueryingInstanceDark from '@img/image/cloud/querying-instance.png';
+
+import OpenInHTTPCurlLight from '@img/image/cloud/open-in-http.png';
+import OpenInHTTPCurlDark from '@img/image/cloud/open-in-http.png';
+
+<Image
+alt="Surreal Cloud (Beta)"
+width={100}
+className="flag-image-title"
+src={{
+	light: LightCloud,
+	dark: DarkCloud,
+	}}  
+/>
+# Deploy on Surreal Cloud (Beta)
+
+## What is Surreal Cloud (Beta)?
+
+Surreal Cloud is a fully managed database service that provides a seamless deployment experience for SurrealDB. It's designed to be lightweight, leveraging the Open-Source SurrealDB product as much as possible. Its features are intended to complement those of the open-source version and offer you a seamless deployment experience.
+
+In this guide, we'll walk through the process and options to deploy SurrealDB using [Surreal Cloud](/docs/cloud). 
+
+## Requirements
+
+To get [started with Surreal Cloud](/docs/cloud/getting-started), you'll need to have an account. If you don't have one, you can sign up for a free account by [downloading Surrealist](https://github.com/surrealdb/surrealist/releases) or using the [Surrealist web app](https://surrealist.app/cloud). 
```

Comment:
How about a comment here that it only takes a minute and you can authorize through something like an existing Google account? That's an easy process many people will be familiar with.